bugfixtemplates/format-int-using-percent-d/FormatIntUsingPercentD.java

```diff
@@ -0,0 +1,48 @@
+package formatint;
+
+import java.io.File;
+
+class FormatIntUsingPercentD {
+
+    int getFileLengthShim(File file) {
+        return (int) file.length();
+    }
+    
+    void formatint_m11(File file) {
+        int i = file.length();
+        String s = String.format("File length is %s", i);
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

This code uses &#39;%s&#39; to format int: i (declared at line 12). This is a potential locale-sensitive handling issue. It might cause errors in the handling and processing of the statement at line: 13. Consider formatting this data with &#39;%d&#39; instead.